src/main/js/components/subcomponents/Order.js

```diff
@@ -2,4 +2,145 @@
 
 /** ----- NPM PACKAGE IMPORTS -----**/
 import React from "react";
-import ReactDOM from "react-dom";
\ No newline at end of file
+import ReactDOM from "react-dom";
+
+export class Order extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {menuItem: [], diningSession: []}; // current menu item and dining session being considered in submitOrdersToDB
+        this.submitOrdersToDB = this.submitOrdersToDB.bind(this);
+        this.requestMenuItem = this.requestMenuItem.bind(this);
+        this.requestDiningSession = this.requestDiningSession.bind(this);
+        this.testSubmit = this.testSubmit.bind(this);
+    }
```

Comment:
### Cleaning
Remove: `testSubmit` & refactor to allow for connection to cart from App.js. 
May need to entirely wipe this depending on implementation of App.js. 
Will wait till completed cart functionality